.gitattributes

```diff
@@ -0,0 +1,20 @@
+# Check out all text files in UNIX format.
+* text eol=lf
+
+# Explicitly declare text files we want to always be normalized and converted 
```

Comment:
@simonjenga I don't thind the lines bellow are needed... git already know what is and what is not a text file...